### PR TITLE
Fixes verbose's boolean values and its unittest

### PR DESCRIPTION
**Pull request by jkpnen**
Fixes conflicts between boolean value initializations which gave unittest errors.

### CODE REVIEW

**Comment by tpitkanen** [comment]
I noticed a bug here:

https://github.com/jkpnen/potku/blob/792cb063d1707cabc96007ecc09af949d7c31968/modules/element_simulation.py#L832

The `False` in `verbose: False,` is currently a type annotation, not a default value. This should fix it: `verbose: bool = False,`.

**Comment by jkpnen** [comment]
> I noticed a bug here:
> 
> https://github.com/jkpnen/potku/blob/792cb063d1707cabc96007ecc09af949d7c31968/modules/element_simulation.py#L832
> 
> The `False` in `verbose: False,` is currently a type annotation, not a default value. This should fix it: `verbose: bool = False,`.

Solved

File: modules/measurement.py [hunk 1]
```diff
@@ -586,6 +585,8 @@ def _measurement_to_file(self, measurement_file: Optional[Path] = None):
         with measurement_file.open("w") as file:
             json.dump(obj_measurement, file, indent=4)
 
+        return measurement_file
```
Comment (tpitkanen) [inline modules/measurement.py:588]: Why is this returned here?

File: ui_files/ui_optimization_fluence_params.ui [hunk 2]
```diff
@@ -326,6 +326,9 @@
                <property name="text">
                 <string/>
                </property>
+               <property name="checked">
+                <bool>true</bool>
```
Comment (tpitkanen) [inline ui_files/ui_optimization_fluence_params.ui:330]: Is this a requested change? I can't find any context (e.g. GitHub issue or commit message) for it.

Comment (jkpnen) [reply ui_files/ui_optimization_fluence_params.ui:330]: It a valid change / [property ](https://doc.qt.io/qt-5/qcheckbox.html#tristate-prop)that comes from Qt Designer. I don't know why it appears here.

File: modules/measurement.py (hunk 1 above)
Comment (jkpnen) [reply modules/measurement.py]: For no reason. Removed it.

**Comment by tpitkanen** [comment]
I [cherry-picked](https://git-scm.com/docs/git-cherry-pick) the main fix from this to #188.

**Pull request closed without merging**